### DISCUSSION TITLE
Idea brainstorm

### DISCUSSION THREAD

**Discussion by DanRowe** [DanRowe/Bitcamp2021#1] [Q&A] [answered]
Let's decide on our idea for bitcamp

**Comment by amknox5** [comment]


**Reply by Jobenland** [reply]
I agree

**Reply by amknox5** [reply]
What do you think we should do Jonathan?

**Comment by Jobenland** [comment]
What do we think about creating something with the google calendar API to create a script that let us auto-generate a text string with our availability?

ETA: **1 hour**
Difficulty: **easy**

**Reply by amknox5** [reply]
I think that would not only be interesting but also useful! I also think the cost of living calculator would also be interesting as well but could be difficult trying to get the information about the leasing prices around cities. I also liked the idea of creating a better when2meet cause I do not like or prefer using the current iterations of it. That would not be too difficult either. Up to you what we want to pursue!

**Reply by Jobenland** [reply]
When to meet would take a bit longer. I would assume that creating a web application as such would be more in the range of about 7 to 8 hours and we may not be able to complete this activity

**Reply by amknox5** [reply]
What would you suggest then? We have some code already for the indeed one if you want to do that one. Hopefully wouldn't be too hard or time-consuming.

**Comment by amknox5** [comment, marked as answer]
https://www.thecocktaildb.com/api.php
This is the cocktial api

**Reply by amknox5** [reply]
https://cocktailsapi.xyz/

**Reply by amknox5** [reply]
import requests
import json
import webbrowser
def cocktails():
    f = r"https://www.thecocktaildb.com/api/json/v1/1/search.php?s=margarita"
    data = requests.get(f)
    tt = json.loads(data.text)
    #webbrowser.open(tt["url"])
    
    for i in (tt["drinks"]):
        print(i, "\n")
    print(len(tt["drinks"]))    
cocktails()

**Reply by amknox5** [reply]
import requests
import json
import webbrowser
def cocktails():
    f = r"https://www.thecocktaildb.com/api/json/v1/1/search.php?s=margarita"
    data = requests.get(f)
    tt = json.loads(data.text)
    
    
    for i in (tt["drinks"]):
        print(i["strDrink"], "\n")
        print(i["strInstructions"], "\n")
        print(i["strInstructionsDE"], "\n")
        
        print(i["strIngredient1"])
        print(i["strIngredient2"])
        print(i["strIngredient3"])
        print(i["strIngredient4"])
        url = i["strDrinkThumb"]
        webbrowser.open(url)
cocktails()
